api/routes/users.js

```diff
@@ -0,0 +1,72 @@
+const express = require("express");
+const User = require("../models/User");
+const Post = require("../models/Post");
+const authenticated = require("../middleware/withAuth");
+
+const app = express();
+
+// Responds with all  users firstName lastName and email
+app.get("/users", authenticated, (req, res) => {
+  User.findAll({
+    attributes: ["firstName", "lastName", "email"],
+  })
+    .then((data) => {
+      console.log(data);
+      res.status(200).send(data);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.sendStatus(400);
+    });
+});
+
+//  Responds with  one person and their posts
+app.get("/users/:id", authenticated, (req, res) => {
+  const user = req.params;
+
+  User.findOne({
+    where: { user_id: user.id },
+    include: [
```

Comment:
Can you add this  add here => `attributes: { exclude: ['password'] }` we should never send this kind of data to client

Reply:
For certainty i can add the exclude attribute. My question is  
I'm not calling it to be shown so how can the user see it?

Reply:
findAll basically =>`SELECT * FROM ...` [more info](https://sequelize.org/master/manual/model-querying-basics.html)

Reply:
Findall looks for all users, if you specifically asked for attributes you want it automatically exclude the others(which are`attributes: ["firstName", "lastName", "email"]...` but to be sure I have added exclude